sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Table.cs

```diff
@@ -125,6 +125,21 @@ private static DynamoDBEntryType GetType(string attributeType)
             throw new InvalidOperationException("Unknown attribute type");
         }
 
+        private static ScalarAttributeType PrimitiveToScalar(DynamoDBEntryType primitiveEntryType)
+        {
+            switch (primitiveEntryType)
+            {
+                case DynamoDBEntryType.String:
+                    return ScalarAttributeType.S;
+                case DynamoDBEntryType.Numeric:
+                    return ScalarAttributeType.N;
+                case DynamoDBEntryType.Binary:
+                    return ScalarAttributeType.B;
+                default:
+                    throw new ArgumentOutOfRangeException($"{primitiveEntryType} is not a known DynamoDB {nameof(ScalarAttributeType)}");
```

Comment:
I believe the single `string` signature is expecting the name of the invalid parameter, not a message.

We probably want something like:

```suggestion
                    throw new ArgumentOutOfRangeException(nameof(primitiveEntryType), $"{primitiveEntryType} is not a known DynamoDB {nameof(ScalarAttributeType)}");
```

Reply:
Fixed